src/components/shared/toggleButton/toggleButton.js

```diff
@@ -1,15 +1,26 @@
 import React from "react";
+import PropTypes from "prop-types";
 import "./toggleButton.scss";
 
-const TogleButton = () => {
+const ToggleButton = ({ children }) => {
   return (
     <div
       type="button"
       className="dropdown-toggle"
       data-toggle="dropdown"
       aria-haspopup="true"
       aria-expanded="false"
-    ></div>
+    >
+      {children}
+    </div>
   );
 };
-export default TogleButton;
+
+ToggleButton.defaultProps = {
+  children: React.createElement("div"),
```

Comment:
Replace with empty fragment to not create extra `div` tag

Reply:
Done